glib/glib.go

```diff
@@ -1278,7 +1280,7 @@ func marshalPointer(p uintptr) (interface{}, error) {
 
 func marshalObject(p uintptr) (interface{}, error) {
 	c := C.g_value_get_object((*C.GValue)(unsafe.Pointer(p)))
-	return newObject((*C.GObject)(c)), nil
+	return NewObject((*C.GObject)(c)), nil
```

Comment:
This should probably also call `Take()`?

Reply:
let's go with a glib.Take() for now